com.microsoft.mrtk.graphicstools.unity/Samples~/Magnifier/Shaders/Magnifier.shader

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Should this shader live in Runtime/Experimental/Magnifier/Shaders since it is required for the magnification feature?